common-components/index.md

```diff
@@ -116,21 +115,51 @@ Read the [Cloud Infrastructure and Platform Services wiki](https://educationgovu
 - Head of DevOps - [Jeffery Odiase](https://eur.delve.office.com/?u=a206fb4f-4f84-4717-b250-16d2577faeaf&v=work)
 - Platform support lead - [Paul Cripwell](https://eur.delve.office.com/?u=11bcb046-7de9-4dd9-858d-7997129df02d&v=work)
 
-
-
 <a name="eapim"></a>
 ### 2. Enterprise API Management
+Enterprise Integration and API management services make it easier to share data quickly and securely – whether you’re an integration partner, developer, publisher, or data consumer.
 
-[description]
+##### Enterprise API Management (EAPIM) Platform
+The EAPIM platform provides a central place to host and discover APIs. It is a secure and fully accredited service that makes it easy to both publish and consume APIs.
 
-__Documentation__
+EAPIM has been developed using Microsoft Azure’s API gateway, and the solution is deployed within the [CIP hosting platform](#cip) with an expert team supporting the infrastructure.
 
-__Support__
+Below is an overview of some of the key benefits.
 
-__Roadmap__
+**Making it easier to share data**
+- Developers can securely publish their APIs quickly in one place using the developer portal
+- Data consumers can view all the APIs available and request access to data
+- The service scales quickly in response to changing demand
 
-__Further guidance__
+**Reducing the admin for developer teams**
+- Developers can host their APIs quickly by using automated features to build, test and publish. Features include automated on-boarding, templates, tested global policy controls, and identity and access management
+- Once an APIs is hosted, there’s no need for developers to perform any platform maintenance
+- Security accreditation is already in place
+
+**Cost savings**
+- EAPIM has a flexible subscription model, which means you won’t pay for more hosting space than you use
+- By moving APIs to EAPIM, we are reducing the number of duplicate API platforms across the estate
+
+##### Integration middleware
```

Comment:
Will need to see how these headings look when published, over just making them bold. Headings get swept up into the side navigation, so might make that a bit chaotic...